lucene/analysis/common/src/test/org/apache/lucene/analysis/hunspell/TestPerformance.java

```diff
@@ -57,12 +57,12 @@ public static void resolveCorpora() {
 
   @Test
   public void en() throws Exception {
-    checkAnalysisPerformance("en", 1_000_000);
+    checkAnalysisPerformance("en", 1_200_000);
   }
 
   @Test
   public void en_suggest() throws Exception {
-    checkSuggestionPerformance("en", 1_200);
+    checkSuggestionPerformance("en", 3_000);
```

Comment:
Hmm what are these magical constant numbers?  And why does this change mean they should increase?

Reply:
These are the number of words to check in the test so that its duration is noticeable enough for profiling, but not too long for me to get bored when running it locally :)